### ISSUE TITLE
Ubuntu az acr build doesn't work after azure-cli upgrade to 2.26.0-1~focal (which is probably what triggerede the problem)

### ISSUE THREAD

**Issue by HansK-p** [Azure/azure-cli#18806]
## Describe the bug

**Command Name**
`az acr build`

**Errors:**
```
The command failed with an unexpected error. Here is the traceback:
'NoneType' object is not callable
Traceback (most recent call last):
  File "/opt/az/lib/python3.6/site-packages/knack/cli.py", line 231, in invoke
    cmd_result = self.invocation.execute(args)
  File "/opt/az/lib/python3.6/site-packages/azure/cli/core/commands/__init__.py", line 657, in execute
    raise ex
  File "/opt/az/lib/python3.6/site-packages/azure/cli/core/commands/__init__.py", line 720, in _run_jobs_serially
    results.append(self._run_job(expanded_arg, cmd_copy))
  File "/opt/az/lib/python3.6/site-packages/azure/cli/core/commands/__init__.py", line 691, in _run_job
    result = cmd_copy(params)
  File "/opt/az/lib/python3.6/site-packages/azure/cli/core/commands/__init__.py", line 328, in __call__
    return self.handler(*args, **kwargs)
  File "/opt/az/lib/python3.6/site-packages/azure/cli/core/commands/command_operation.py", line 121, in handler
    return op(**command_args)
  File "/opt/az/lib/python3.6/site-packages/azure/cli/command_modules/acr/build.py", line 109, in acr_build
    variant=platform_variant
TypeError: 'NoneType' object is not callable

```

## To Reproduce:
Steps to reproduce the behavior. Note that argument values have been redacted, as they may contain sensitive information.

- _Put any pre-requisite steps here..._
- `az acr build --registry {} --subscription {} --platform {} --image {} --file {} {}`

## Expected Behavior
Expected az acr build to work

## Environment Summary
```
Linux-5.4.0-77-generic-x86_64-with-debian-bullseye-sid
Python 3.6.10
Installer: DEB

azure-cli 2.26.0

```
## Additional information
I downgraded azcli as a workaround and that "solved" the problem, but it is of course not a permament solution. I have the same issue on two relatively independent machines:
<pre>
dpkg: warning: downgrading azure-cli from 2.26.0-1~focal to 2.24.2-1~focal
</pre>
Version 2.24.2-1~focal was chosen as I have a debian server with 2.24.2 active and it have been working all the time.

**Comment by codycodes** [comment]
I'm sure this will be resolved soon, but if you need to workaround the issue here are the steps I took:

If you're on Windows, you can most likely just install the earlier [release](https://github.com/Azure/azure-cli/releases)

For those like me who are using WSL or other *nix-based OS and are using `apt` as your package manager, here are the steps I took to downgrade:

```
# Use apt-cache to install different versions of a package

sudo apt-get update

# get the available versions using: apt-cache showpkg <package-name>
# install a specific version using: sudo apt-get install package=version -V

sudo apt-get install azure-cli=2.23.0-1~focal -V
```

More info: [How to Install Specific Version of Package using apt-get (linoxide.com)](https://linoxide.com/install-specific-version-package-apt-get/)

**Comment by yonzhan** [comment]
route to service team

**Comment by northtyphoon** [comment]
The issue has been fixed since 2.26.1
